src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  static removeDeadAnimals() {
+    this.alive = this.alive.filter(animal => animal.health < 0);
```

Comment:
Now you'll have all animals removed from `alive`.
Correct filter should return true for all animals with `health > 0`
```suggestion
    this.alive = this.alive.filter(animal => animal.health > 0);
```

Reply:
hahahah maaan, how in the world did i make that mistake 🤣

Reply:
Funniest thing is that tests still pass 🤣